extensions/Needinfo/Extension.pm

```diff
@@ -39,8 +39,6 @@ sub install_update_db {
     print "Creating needinfo flag ... " . 
           "enable the Needinfo feature by editing the flag's properties.\n";
 
-    # Initially populate the list of exclusions as __Any__:__Any__ to
```

Comment:
This comment is removed but not replaced, leaving the weird ```['0:0']``` inclusion less-documented than the exclusion it replaces.